.github/workflows/integration.yml

```diff
@@ -74,8 +74,8 @@ jobs:
       - name: Build all assets
         run: make assets
 
-      - name: Change MySQL authentication method
-        run: mysql -h127.0.0.1 -uroot -ppassword -e "ALTER USER 'root'@'%' IDENTIFIED WITH mysql_native_password BY 'password'; FLUSH PRIVILEGES;"
+      - name: Change MariaDB authentication method
+        run: mysql -h127.0.0.1 -uroot -ppassword -e "ALTER USER root@localhost IDENTIFIED VIA mysql_native_password USING PASSWORD('password'); FLUSH PRIVILEGES;"
```

Comment:
This chunk isn't needed. The `mysql root password` of the `getlong/mariadb-action` will set the password. Also `FLUSH PRIVILEGES isn't required after any ALTER USER or other SQL statement granting privs/user changes.